test/e2e/common/BUILD

```diff
@@ -79,7 +79,6 @@ go_library(
         "//staging/src/k8s.io/client-go/tools/watch:go_default_library",
         "//test/e2e/framework:go_default_library",
         "//test/e2e/framework/kubelet:go_default_library",
-        "//test/e2e/framework/log:go_default_library",
```

Comment:
The latest code doesn't contain this dependency as https://github.com/kubernetes/kubernetes/blob/master/test/e2e/common/BUILD#L81

So I think this PR becomes today.